packages/flutter_tools/lib/src/macos/build_macos.dart

```diff
@@ -14,6 +14,8 @@ import '../ios/xcodeproj.dart';
 import '../project.dart';
 import 'cocoapod_utils.dart';
 
+final RegExp _anyOutput = RegExp('.*');
```

Comment:
I think it's worth commenting why this is done (i.e., the interaction with `-quiet`) since it's non-obvious without context.

Reply:
Done